### ISSUE TITLE
property name mismatch in onRendered function

### ISSUE THREAD

**Issue by jaraquistain** [jhollingworth/marty-express#10]
I'm currently migrating to marty 0.10.0 and am running into an error when trying to server up my base route.

Here's what my usage of the middleware looks like: 

``` javascript
import express from "express";
import martyExpress from "marty-express";
import routes from "./config/Routes";
import application from "./app";

let server = express();

server.use(martyExpress({
    routes,
    application
}));
```

Once the server is running going to / (or any other route) results in the following error:

```
Failed to render / TypeError: Cannot read property 'trim' of undefined
    at onRendered (~/Development/site/node_modules/marty-express/index.js:92:63)
    at runMicrotasksCallback (node.js:337:7)
    at process._tickCallback (node.js:355:11)
```

looking at the offending line yields the following:

``` javascript
locals[options.body || 'body'] = renderResult.htmlBody.trim();
locals[options.state || 'state'] = renderResult.htmlState.trim();
```

and logging out `renderResult` shows an object which contains only 2 properties: `html` and `diagnostics`.

The documentation for the migration of renderToString from Marty to app mentions `html` and `diagnostics` but not `htmlBody` nor `htmlState`

``` javascript
app.renderToString(<Foo bar='baz' />, { timeout: 2000 }).then(res => {
  console.log('Rendered html', res.html);
  console.log('Diagnostics', res.diagnostics);
});
```

Is this a problem or am I doing something wrong?

**Comment by frasermince** [comment]
I'm getting this same error. It seems to be happening with me on the pending or failed state. At the very least these do not seem to have a htmlBody element. I have no idea if the success state does or not. I haven't gotten that far.

**Comment by jhollingworth** [comment]
Sorry about this. I'd done a final patch for v0.9 to resolve martyjs/marty#288 but it hadn't properly made it over to v0.10. I've just released Marty v0.10.1 which should add in `htmlBody` and `htmlState`

**Comment by jhollingworth** [comment]
the upgrade guide as also been updated http://martyjs.org/guides/upgrading/09_10.html#isomorphism
